db/seeds.rb

```diff
@@ -332,7 +332,9 @@ def setup_hmis_admin_access
   return unless user.present?
 
   user.hmis_data_source_id = hmis_ds.id
-  user.user_hmis_data_sources_roles.create(role: role, data_source_id: user.hmis_data_source_id)
+  access_group = Hmis::AccessGroup.create!(name: 'test')
```

Comment:
I don't _think_ we want to do this here without some guard on when it gets called.  `db:seed` is called on every deploy, and we don't want a test access group installed for production.
If this is used for testing, it could just get wrapped in an `if Rails.env.test?` or move to a factory.

Reply:
@eanders - Thanks, I didn't realize we were running this on deploy. I added some documentation for this to the top of the seeds file.

I noticed that `user.user_hmis_data_sources_roles` does not exist. I'm surprised this doesn't generate an exception on deploy. Perhaps this line should just be removed?

Reply:
@gigxz any thoughts on `user.user_hmis_data_sources_roles`, I'm not familiar with that one.  Maybe that's obsolete after @ZachCMP 's recent upgrade to ACLs?  

Thanks @ttoomey, I think this looks good.

Reply:
thanks @eanders - my understanding is that I should rebase this into r70 once the sprockets issue is resolved.

Reply:
@ttoomey you can give https://github.com/greenriver/hmis-warehouse/pull/2910 a try and see if that solves it for you.  (It should.)  Otherwise, it's totally fine to wait until that gets merged into `release-70`

Reply:
Yeah, I have some code in my local tasks for seeding the HMIS admin roles. I'll go back and update this file to be correct. We shouldn't be auto-assigning HMIS admin perms to a random user, so I'll remove that part.